internal/db/rdb/mysql/mysql.go

```diff
@@ -175,41 +174,41 @@ func (m *mySQLClient) Close(ctx context.Context) error {
 	return nil
 }
 
-// GetMeta gets the metadata and podIPs which have index of metadata's vector.
-func (m *mySQLClient) GetMeta(ctx context.Context, uuid string) (MetaVector, error) {
+// GetVector gets the vecor adata and podIPs which have index of vector data's vector.
```

Comment:
typo `vecor adata`

Reply:
`vector data's vector` is too much. I think `vector` is enough.

Reply:
Could you please use github suggestion for each comment ?